tests/test_makefile.mak

```diff
@@ -20,23 +20,22 @@ APP_NAME =
 
 SHARED_CODE = ../../shared_src
 
-COMMON_FLAGS = -g -report -DDEBUG_PRINT_ENABLE -save-temps -O3 -Xmapper --map -Xmapper MAPFILE -I$(SHARED_CODE) -DUSB_TILE=tile[0] -DSIMULATION -DARCH_L
+TEST_FLAGS ?=
 
-XCC_FLAGS_xs2       = $(COMMON_FLAGS) -DARCH_X200 -DXUD_SERIES_SUPPORT=XUD_X200_SERIES
+COMMON_FLAGS = -g -report -DDEBUG_PRINT_ENABLE -save-temps -O3 -Xmapper --map -Xmapper MAPFILE -I$(SHARED_CODE) \
+			   -DUSB_TILE=tile[0] \
+			   -Wno-timing \
+			   -DXUD_SIM_XSIM=1 \
+			   -DXUD_TEST_SPEED_HS=1 \
+			   $(CFLAGS)
 
-XCC_FLAGS_xs1       = $(COMMON_FLAGS) -DARCH_S -DXUD_SERIES_SUPPORT=XUD_U_SERIES
-
-
-
-ifeq ($(CONFIG),$(filter $(CONFIG),xs1))
-	TARGET = test_xs1.xn
-endif
-
-ifeq ($(CONFIG),$(filter $(CONFIG),xs2))
-	TARGET = test.xn
-endif
+#TODO RM ARCH_L define
+#XCC_FLAGS_xs2       = $(TEST_FLAGS) $(COMMON_FLAGS) -DXUD_SERIES_SUPPORT=XUD_X200_SERIES -DARCH_L
```

Comment:
Does this Make file include support for testing on XS2?  The `XCC_FLAGS_xs2` symbol has been commented out but it still defines `XUD_SERIES_SUPPORT` as `XUD_X200_SERIES`.  Isn't the definition of `ARCH_L` redundant?

Reply:
I will remove completely for now